bundles/AdminBundle/src/Controller/Admin/SettingsController.php

```diff
@@ -528,6 +529,10 @@ public function setSystemAction(
         $settingsYaml = Yaml::dump($settings, 5);
         $configFile = Config::locateConfigFile('system.yaml');
         File::put($configFile, $settingsYaml);
+        if ($existingValues) {
+            $saveSettingsEvent = new GenericEvent(null, [$existingValues['pimcore']['general']['valid_languages'], $filteredLanguages]);
```

Comment:
Shouldn't we include all settings values here? At least the name of the event is generic, so you'd expect not just the language values here, right?

Reply:
That is a good point, thank you. It should be passing all settings values now.